src/components/ProjectMembersSummary/ProjectMembersSummary.tsx

```diff
@@ -0,0 +1,37 @@
+import { Group } from '@material-ui/icons';
+import { isEmpty } from 'lodash';
+import { FC } from 'react';
+import * as React from 'react';
+import { MemberListSummary, MemberSummaryItem } from '../MemberListSummary';
+import { randomPersonPic } from '../MemberListSummary/MemberListSummary.stories';
+import { ProjectMemberListFragment } from './ProjectMembersSummary.generated';
+
+export interface ProjectMembersSummaryProps {
+  memberList?: ProjectMemberListFragment;
+}
+
+export const ProjectMembersSummary: FC<ProjectMembersSummaryProps> = ({
+  memberList,
+}) => {
+  const members: MemberSummaryItem[] | undefined = memberList?.items
+    ?.filter(({ user }) => !isEmpty(user.value?.displayFirstName?.value))
+    .map(({ user }) => ({
+      // TODO is this like card where displayFirstName is used if real first name doesn't exist
+      picture: randomPersonPic(
+        user.value?.displayFirstName?.value ?? '',
+        'male'
+      ),
+      avatarLetters: user.value?.displayFirstName?.value ?? '',
+      label: user.value?.displayFirstName?.value ?? '',
+    }));
+
+  // TODO what to use for path
+  return (
+    <MemberListSummary
+      title="Team Members"
+      members={members}
+      to="/teamMembers"
```

Comment:
urls are always _kebab-case_. We also don't want to use an absolute path (starting with `/`) as that wouldn't prefix the current location (the specific project). _team_ & _members_ mean the same thing. Let's try to be consistent with the API instead of UI.
```suggestion
      to="members"
```

Reply:
Okay thanks, will use kebab case and look at api to determine url from now on.